src/corebluetooth.cs

```diff
@@ -34,9 +34,17 @@ interface CBCentralInitOptions {
 #endif
 	}
 
+	[iOS (10,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface CBManager {
+		[Export ("state", ArgumentSemantic.Assign)]
+		CBManagerState State { get; }
+	}
+
 	[Since (5,0)]
 	[Lion]
-	[BaseType (typeof (NSObject), Delegates=new[] {"WeakDelegate"}, Events = new[] { typeof (CBCentralManagerDelegate)})]
+	[BaseType (typeof (CBManager), Delegates=new[] {"WeakDelegate"}, Events = new[] { typeof (CBCentralManagerDelegate)})]
 	[DisableDefaultCtor] // crash (at dispose time) on OSX
 	interface CBCentralManager {
 		[Export ("state")]
```

Comment:
Well it looks like they did not (yet) remove the `state` property from `CBCentralManager.h` on macOS. Should I `!XAMCORE_4_0` in advance?